assets/javascript/app.js

```diff
@@ -0,0 +1,178 @@
+    answers: ["0%", "25%", "50%", "75%"],
+    correctAnswer: "75%",
+    image: "assets/images/bathroom.jpg"
+} , {
+    question: "Aaaaand somewhat predictably, ___ million phones a year fall in. SPLOOOOSH!!!",
+    answers: ["7", "100", "2", "40"],
+    correctAnswer: "7",
+    image: "assets/images/phoneintoilet.jpg"
+} , {
+    question: "Americans are really on a roll… we use ____ million miles of toilet paper every year – that’s enough to stretch to the sun and back!",
+    answers: ["100", "50", "900", "433"],
+    correctAnswer: "433",
+    image: "assets/images/toiletpaperrolls.jpg"
+} , {
+    question: "When it comes to gender, men and women have different bathroom pet peeves. Men’s top annoyances are makeup on the towels and _________________.",
+    answers: ["leaving the toilet seat down", "hair in the sink", "hair in the shower", "toothpaste in the sink"],
+    correctAnswer: "hair in the sink",
+    image: "assets/images/hairinsink.jpg"
+} , {
+    question: "The top complaints for women were leaving the seat up (who didn’t see that one coming?), and _________________.",
+    answers: ["leaving the toilet seat down", "hair in the sink", "hair in the shower", "toothpaste in the sink"],
+    correctAnswer: "toothpaste in the sink",
+    image: "assets/images/toothpasteinsink.jpg"
+} , {
+    question: "And in defense of women everywhere, putting the seat down just makes good safety sense. A full ____ of bathroom-related injuries happen when someone falls into the toilet because the seat has been left up!.",
+    answers: ["10%", "30%", "75%", "85%"],
+    correctAnswer: "85%",
+    image: "assets/images/toiletfall.jpg"
+}];
+
+var game = {
+    questions:questions,
+    currentQuestion:0,
+    counter:30,
+    correct:0,
+    incorrect:0,
+    unanswered:0,
+    countdown: function() {
+        // Countdown from 30 seconds
+        game.counter--;
+        // Update to HTML
+        $("#counter").html(game.counter);
+        // Time's up if less than 1 second
+        if (game.counter < 1) {
+            console.log("Time's Up!");
+            game.timeUp();
+        }
+    }, 
+    loadQuestion: function() {
+        // Countdown one number per second
+        timer = setInterval(game.countdown, 1000);
+        $('#subwrapper').html("<h2>Time Remaining: <span id='counter'>30</span></h2>")
+        // Add header for question
+        $("#subwrapper").append("<h2>" + questions[game.currentQuestion].question + "</h2>");
+        // Load the question while adding a data-name to store the question
+        for (var i = 0; i < questions[game.currentQuestion].answers.length; i++) {
+            $("#subwrapper").append('<button class="answer-button" id="button-' + i + '" data-name="' + questions[game.currentQuestion].answers[i] + '">' + questions[game.currentQuestion].answers[i] + '</button>');
```

Comment:
It's fine creating the button this way, but you can always create variables with the `<button>` and dynamically add the class and data in it with jquery just in case you want to use the seperate variables later